src/server/services/image.service.ts

```diff
@@ -826,6 +827,15 @@ export const getAllImages = async ({
   //   )`);
   // }
 
+  if (baseModels?.length) {
+    AND.push(Prisma.sql`i.id IN (
+      SELECT ir."imageId"
+      FROM "ImageResource" ir
+      JOIN "ModelVersion" mv ON ir."modelVersionId" = mv.id
+      WHERE mv."baseModel" IN (${Prisma.join(baseModels)})
+    )`);
+  }
+
```

Comment:
This looks like it could be quite expensive to run - I'd seek approval from Koen/Sean firsrt. Image resources is at least as big as images afaik (far more I think) so it could be quite expensive to look into all records of a base model (i.e, I expect 1.5 to have quite the number of images). On dev the max ID of that table is `27,904,548`